scripts/install.sh

```diff
@@ -0,0 +1,368 @@
+#!/bin/sh
+set -e
```

Comment:
**Nit:** In general I'd prefer using long options when available in scripts that are saved to a file. It helps with readability, especially for those less practiced in the shell.

As an example, instead of `set -e` I would prefer `set -o errexit`.

Reply:
Makes sense, changed to set -o errexit